### PR TITLE
Assorted fixes

### PR DESCRIPTION
**Pull request by fjork3**
- None as valid return type on normalizers
- typo/docstring cleanup
- clean unused imports
- missing `await` on async call
- improper runner call in API init

### CODE REVIEW

File: src/scooze/database/core.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ async def insert_document(coll_type: DbCollection, document: dict[str, Any]):
     """
 
     # Here we find and update with upsert=True instead of inserting to avoid creating duplicates in the database. This
-    # creates less headaches if you doubleclick in the Swagger UI or similar.
+    # creates fewer headaches if you doubleclick in the Swagger UI or similar.
```
Comment (JacobGinsparg) [inline src/scooze/database/core.py:24]: Me realizing I made a grammar mistake

image

Comment (fjork3) [reply src/scooze/database/core.py:24]: image

File: src/scooze/utils.py [hunk 2]
```diff
@@ -333,7 +333,7 @@ def to_date(cls, d: date | str | None) -> date:
         return datetime.strptime(d, "%Y-%m-%d").date()  # NOTE: maybe store date format
 
     @classmethod
-    def to_enum(cls, e: Type[E], v) -> E:
+    def to_enum(cls, e: Type[E], v) -> E | None:
```
Comment (JacobGinsparg) [inline src/scooze/utils.py:336]: The `|` notation is a Python 3.10+ thing afaik. Do we want to track somewhere things we know we'll have to change to support 3.9 and earlier? Idk if we have a doc going already.

**Review by JacobGinsparg — DISMISSED** [review]

File: src/scooze/utils.py (hunk 2 above)
Comment (fjork3) [reply src/scooze/utils.py:336]: Made an issue for tracking/discussion: #191

**Review by JacobGinsparg — APPROVED** [review]

File: src/scooze/api/card.py [hunk 3]
```diff
@@ -43,6 +42,7 @@ async def get_cards_by(
     Args:
         property_name: The property to check.
         values: A list of values to match on.
+        card_class: The type of card object to return.
         paginated: Whether to paginate the results.
```
Comment (iambroadband) [inline src/scooze/api/card.py:46]: are these other args present in the function header? I don't see them here.

**Review by iambroadband — APPROVED** [review]

File: src/scooze/api/card.py (hunk 3 above)
Comment (fjork3) [reply src/scooze/api/card.py:46]: I'm seeing 6 args in the function header for `get_cards_by`, and with this change 6 in the docstring underneath.

**Pull request merged**